kernel/tests/s10_benchmarks.rs

```diff
@@ -487,13 +487,13 @@ fn s10_fxmark_benchmark() {
                 if cfg!(feature = "smoke") {
                     cmdline = cmdline.memory(8192);
                 } else {
-                    cmdline = cmdline.memory(core::cmp::max(73728, cores * 2048));
+                    cmdline = cmdline.memory(core::cmp::max(87728, cores * 2048));
                 }
 
                 if cfg!(feature = "smoke") && cores > 2 {
-                    cmdline = cmdline.nodes(2);
+                    cmdline = cmdline.nodes(std::cmp::max(machine.max_cores() / 16, 1));
                 } else {
-                    cmdline = cmdline.nodes(machine.max_numa_nodes());
+                    cmdline = cmdline.nodes(std::cmp::max(machine.max_cores() / 16, machine.max_numa_nodes()));
```

Comment:
Are we assuming max 16 cores per numa node? skylake4x has 24, so this may not be a solid assumption?